ludwig/hyperopt/execution.py

```diff
@@ -585,6 +588,101 @@ def execute(
         return hyperopt_results
 
 
+class RayTuneExecutor(HyperoptExecutor):
+    def __init__(self, hyperopt_sampler, output_feature, metric, split, goal, **kwargs):
+        HyperoptExecutor.__init__(self, hyperopt_sampler, output_feature,
+                                  metric, split)
+        self.search_space = hyperopt_sampler
+        self.output_feature = output_feature
+        self.metric = metric
+        self.split = split
+        self.goal = goal
+        self.trial_id = 0
+
+    def _run_experiment(self, config, hyperopt_dict):
+
+        trial_id = tune.get_trial_id()
+        modified_config = substitute_parameters(
+            copy.deepcopy(hyperopt_dict["config"]), config)
+        hyperopt_dict["config"] = modified_config
+        hyperopt_dict["experiment_name"] = f'{hyperopt_dict["experiment_name"]}_{trial_id}'
+
+        train_stats, eval_stats = run_experiment(**hyperopt_dict)
+        metric_score = self.get_metric_score(eval_stats)
+
+        tune.report(parameters=str(config), metric_score=metric_score,
```

Comment:
hmm Tune already automatically logs the config

Reply:
So you are suggesting to remove `parameters=str(config)`?

Reply:
@richardliaw The intention here was to emulate the results of other `HyperoptExecutors` with the key `parameters` to have a similar interface for the further steps. 
@w4nderlust

Reply:
@w4nderlust yeah, though I guess it doesn't hurt to keep it (as @kaushikb11 suggests that there's some later steps that require it)